### PR TITLE
Try to fix backend

### PR DESCRIPTION
**Pull request by shivamchawla29june**

### CODE REVIEW

File: src/containers/landlord/listingInfo.jsx [hunk 1]
```diff
@@ -1,88 +1,121 @@
...
-            console.log(currentListing);
-        }
+  const handleClick = () => {
+    if (editState === true) {
+      setEditState(false);
+      console.log("success");
+    } else {
+      setEditState(true);
+      console.log("success2");
+      console.log(currentListing);
```
Comment (jessicamecht) [inline src/containers/landlord/listingInfo.jsx:20]: can remove log statements

File: src/containers/landlord/listings.jsx [hunk 2]
```diff
@@ -1,33 +1,45 @@
...
+        )
+        .then((response) => {
+          setDataFromDB(response.data.data);
+          setFetchDataFromDB(false);
+        });
+    }
+  }, [fetchDataFromDB, landlordEmail]);
+
+  // console.log(dataFromDB);
+  const listingData = [];
```
Comment (jessicamecht) [inline src/containers/landlord/listings.jsx:28]: same here

**Review by jessicamecht — APPROVED** [review]
looks good to me

File: src/components/Header.tsx [hunk 3]
```diff
@@ -34,13 +37,22 @@ const Header: React.FunctionComponent<HeaderProps> = ({
 
   const search = () => {
     //TODO add backend call
+    const searchString = `location=${searchInput}&price=${price}&num_bedrooms=${noBed}&num_bathrooms=${noBath}`;
+    axios
+      .get(`http://18.196.64.140:8080/listings/search?${searchString}`)
+      .then((response: any) => {
+        setDataFromDB(response.data.data);
+      });
```
Comment (jessicamecht) [inline src/components/Header.tsx:45]: do we need unit tests for the backend calls?

**Review by jessicamecht — APPROVED** [review]

File: src/containers/landlord/listingInfo.jsx (hunk 1 above)
Comment (shivamchawla29june) [reply src/containers/landlord/listingInfo.jsx:20]: This is coming from the main branch. I think @ssreepathy added this.

File: src/containers/landlord/listings.jsx (hunk 2 above)
Comment (shivamchawla29june) [reply src/containers/landlord/listings.jsx:28]: Got it

**Pull request merged**